orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/job/RunJobTask.groovy

```diff
@@ -75,9 +75,14 @@ class RunJobTask extends AbstractCloudProviderAwareTask implements RetryableTask
         "notification.type"   : "runjob",
         "kato.result.expected": creator.katoResultExpected,
         "kato.last.task.id"   : taskId,
-        "deploy.account.name" : credentials,
+        "deploy.account.name" : credentials
     ]
 
+    Optional<Long> actualJobTimeout = creator.getJobTimeoutMs(stage)
+    if (actualJobTimeout != null) {
```

Comment:
unless there is some groovy magic at play here I think you want actualJobTimeout.isPresent() instead of a null check (and creator.getJobTimeoutMs should ideally never return null)

Reply:
thanks!! im confused how did it work... fixing!

Reply:
+1